hopsworks-api/src/main/java/io/hops/hopsworks/api/kafka/KafkaService.java

```diff
@@ -363,13 +312,13 @@ public Response ValidateSchemaForTopics(SchemaDTO schemaData) throws KafkaExcept
     switch (kafkaFacade.schemaBackwardCompatibility(schemaData)) {
       case INVALID:
         json.setErrorMsg("schema is invalid");
-        return noCacheResponse.getNoCacheResponseBuilder(Response.Status.NOT_ACCEPTABLE).entity(json).build();
+        return Response.status(Response.Status.NOT_ACCEPTABLE).entity(json).build();
```

Comment:
@kai-chi we should throw a REST KafkaException and let the throwablemapper handle logging and sending the response.

Reply:
@tkakantousis, this will be refactored in the schema PR. This change was simply done to get rid of noCacheResponse but changing it further is not  topics-related

Reply:
Thanks